packages/components/src/internal/components/editable/utils.ts

```diff
@@ -108,8 +108,9 @@ const CELL_KEY_SEPARATOR = '&&';
 export function genCellKeyPrefix(fieldKey: string): string {
     return fieldKey.toLowerCase() + CELL_KEY_SEPARATOR;
 }
-export function genCellKey(fieldKey: string, rowIdx: number, prefixFieldKey?: string): string {
-    return (prefixFieldKey ? prefixFieldKey.toLowerCase() + '/' : '') + genCellKeyPrefix(fieldKey) + rowIdx;
+
+export function genCellKey(fieldKey: string, rowIdx: number): string {
+    return genCellKeyPrefix(fieldKey) + rowIdx;
```

Comment:
I don't think we need to keep `genCellKeyPrefix` any longer either, it can just be merged into `genCellKey`

Reply:
There is one usage of the `genCellKeyPrefix` helper in `EditorModel.getValuesForColumn` to help get all of the cell values for a given column (i.e. cell key prefix).